src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -217,9 +238,11 @@ public boolean apply(Request<?> request) {
         // If the request is uncacheable, skip the cache queue and go straight to the network.
         if (!request.shouldCache()) {
             mNetworkQueue.add(request);
+            sendRequestEvent(request, RequestEvent.REQUEST_QUEUED);
```

Comment:
this feels like it should be done next to request.addMarker("add-to-queue") once.

Reply:
Done